index.js

```diff
@@ -720,8 +1224,15 @@
       });
     };
 
+    //  valueTypes :: [a] -> [Type]
```

Comment:
The argument's type should be `[Any]` rather than `[a]`, since the values needn't all be members of a common type.